src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/EmployerLedChangeOfProviderRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Authorization.ModelBinding;
+using System;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+{
+    public class EmployerLedChangeOfProviderRequest : IAuthorizationContextModel
+    {
+        [FromRoute]
+        public string AccountHashedId { get; set; }
+        [FromRoute]
+        public string ApprenticeshipHashedId { get; set; }
+
+        public long ApprenticeshipId { get; set; }
+        public long ProviderId { get; set; }
+        public string ProviderName { get; set; }
+
+        public int? NewStartMonth { get; set; }
+        public int? NewStartYear { get; set; }
+        public int? NewEndMonth { get; set; }
+        public int? NewEndYear { get; set; }
+        public int? NewPrice { get; set; }       
+        public bool? IsEdit { get; set; }
```

Comment:
Is this flag required

Reply:
CON-2518, CON-2519, CON-2520 - we have decided together to use this Model, can change it if required